### ISSUE TITLE
Disabling "Forward emails to ---" option, user should get a warning 

### ISSUE THREAD

**Issue by mMARUF** [mozilla/fx-private-relay#307]
### **Steps to Reproduce;**

1. Create an alias from Firefox Relay
2. Create a Twitter account using that alias (**disable** the Forward emails to ******* -> option)

**Actual Results**
There is no warning/tooltip while disabling the "Forward emails to ---" option

**Expected Results**
In many services/website while we sign up, we get a confirmation code to complete signing up process. Thus, when a user, disables the  **Forward emails to ********* option, the user will not get a verification/confirmation code into the actual email and  thus the sign up will not be completed. That is why it is vital in the Firefox relay website, when a user disables the "Forward emails to ---" option, user should get a warning saying that **"You will not get confirmation codes/updates/notifications if you disable this option"**

**Comment by lesleyjanenorton** [comment]
Agreed. I think we also need to convey that choosing to not forward emails means those emails will be lost and irretrievable (as opposed to being held in a queue of some sort).

**Comment by hackel** [comment]
It would be even nicer if Relay held on to those messages for a limited period of time, similar to 10minutemail-type services.

**Comment by EMMLynch** [comment]
This issue is being automatically closed as it is older than 2 years and has not been addressed. If this issue is still relevant, please feel free to reopen it with updated information.
